tests/test_workers/test_pkg_managers/test_general_js.py

```diff
@@ -42,6 +42,7 @@ def test_download_dependencies(
         angular-devkit-architect-0.803.26.tgz
         angular-animations-8.2.14.tgz
         rxjs-6.5.5-external-gitcommit-78032157f5c1655436829017bbda787565b48c30.tgz
+        exsp-external-archive-2.10.2.tar.gz
```

Comment:
This should be named like this right?
```suggestion
        exsp-2.10.2-external-sha512-abcdefg.tar.gz
```